http4s-scala-fx/src/test/scala/http4s/fx/HttpServerFixtures.scala

```diff
@@ -0,0 +1,38 @@
+package http4s
+package fx
+
+import _root_.fx.{handle, structured, toEither, Control, Resource}
+import _root_.fx.instances.FxAsync.asyncInstance
+import _root_.fx.instances.StructuredF
+import _root_.fx.HttpExecutionException
+import fx.BlazeServerFXBackend
+import munit.FunSuite
+import org.http4s.Method.{GET, POST}
+import org.http4s.{HttpApp, HttpRoutes, Response}
+
```

Comment:
```suggestion
import _root_.fx.Structured

```
Necessary import for below suggestion.